badger/static/css/badger.css

```diff
@@ -35,71 +40,32 @@
 }
 
 
-.color .red    {background: #fa5a5a;}
-.color .yellow {background: #f0d264;}
-.color .green  {background: #82c8a0;}
-.color .cyan   {background: #7fccde;}
-.color .blue   {background: #6698cb;}
-.color .purple {background: #cb99c5;}
-
-pre {
-	width: 100%;
-	padding: 30px;
-	background-color: rgba(0, 0, 0, 0.72);
-	color: #f8f8f2;
-	border-radius: 0 0 4px 4px;
-	margin-top: 20px;
-  white-space: pre-wrap; /* css-3 */
-  white-space: -moz-pre-wrap; /* Mozilla, since 1999 */
-  white-space: -pre-wrap; /* Opera 4-6 */
-  white-space: -o-pre-wrap; /* Opera 7 */
-  word-wrap: break-word; /* Internet Explorer 5.5+ */
+.btn-default-badger {
+    background: #0075b4;
+    color: #fff !important;
+    transition: all 0.3s;
 }
-pre .bt  {color: #f8f8f2;} /* <> */
-pre .anc {color: #f92672;} /* anchor tag */
-pre .att {color: #a6a926;} /* attribute */
-pre .val {color: #e6db74;} /* value */
 
-.btn-gradient {
-	margin: 5px;
+.btn-default-badger:hover, .btn-default:focus {
+    background: #009ade;
+    color: #fff !important;
```

Comment:
please chat to @bronsondunbar about this important, it may cause overrides to fail in the future.
here and on line 45.

Reply:
@nicovanniekerk, I have removed the important so that the overrides can work and fixed the typo. Please let me know if this can be merged so I can update the version number and add the tag as well/